mantidproject/fitbenchmarking
# Changed the default results folder location

#### Description of Work

Fixes #84 
The default results folder location is now in the same directory as the script that was executed.


#### Testing Instructions

1. Run `example_runScript.py` in the mantidpython console.
2. Check if a `results` and `logs` folder are produced and contain what expected files.
